x/campaign/types/campaign.go

```diff
@@ -1,10 +1,10 @@
 package types
 
 import (
-	"errors"
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/pkg/errors"
```

Comment:
was this dependency substitution intended?

Reply:
`github.com/pkg/errors` provides the `errors.Wrap` utility